src/tools/gophish_import.py

```diff
@@ -202,29 +208,33 @@ def build_campaigns(api, assessment):
                 break
             except Error as e:
                 if e.message == "SMTP name already in use":
-                    logging.warning(f"{e}. Finding previously loaded smtp to delete.")
+                    logging.warning("%s. Finding previously loaded smtp to delete.", e)
                     smtps = api.smtp.get()
                     logging.debug(
-                        f"Checking {len(smtps)} for previously imported smtp profiles to get ID"
+                        "Checking %i for previously imported smtp profiles to get ID",
+                        len(smtps),
                     )
                     for old_smtp in smtps:
                         if old_smtp.name == new_smtp.name:
-                            logging.debug(f"Deleting SMTP with ID {old_smtp.id}")
+                            logging.debug("Deleting SMTP with ID %i", old_smtp.id)
```

Comment:
Same question as above about `%d` or `%s`:
```suggestion
                            logging.debug("Deleting SMTP with ID %d", old_smtp.id)
```

Reply:
See aa93585